docs/UserGuide.md

```diff
@@ -233,6 +233,13 @@ Furthermore, certain edits can cause the AddressBook to behave in unexpected way
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
```

Comment:
Could this be improved? Maybe talking about what data file it refers to, or where to transfer it?